code/places-monitor-android/src/test/java/com/adobe/marketing/mobile/PlacesMonitorInternalTests.java

```diff
@@ -890,55 +890,6 @@ public void test_getPOIsForLocation_when_failure_with_OtherErrors() {
 		verify(geofenceManager, times(0)).stopMonitoringFences(anyBoolean());
 	}
 
-	// ========================================================================================
-	// updateLocationAuthorizationStatus
-	// ========================================================================================
-	@Test
-	public void test_UpdateLocationAuthorizationStatus_when_always_then_sentToPlacesExtension() {
-		// setup
-		initWithContext(context);
-		verifyStatic(Places.class, times(1)); // once for initialization
-		Places.setAuthorizationStatus(PlacesAuthorizationStatus.ALWAYS);
-		PowerMockito.stub(PowerMockito.method(PlacesActivity.class, "isBackgroundPermissionGranted")).toReturn(true);
-
-		// test
-		monitorInternal.updateLocationAuthorizationStatus();
-
-		// verify
-		verifyStatic(Places.class, times(2)); // once after the call
-		Places.setAuthorizationStatus(PlacesAuthorizationStatus.ALWAYS);
-	}
-
-	@Test
```

Comment:
why did we delete these tests?  we should fix the tests instead of deleting them :)

Reply:
I removed this method from the PlacesMonitor Internal class
https://github.com/adobe/places-monitor-android/pull/31/files#diff-55f93e146e6fde4b18f63a292c7c163fL403

and then put them in PlacesAcitvity class
https://github.com/adobe/places-monitor-android/pull/31/files#diff-33409094082b7619e23d3ebc8d14a4eaR50

And I moved the tests as well :))